pkg/compiler/function_call_test.go

```diff
@@ -121,7 +121,7 @@ func TestFunctionWithVoidReturn(t *testing.T) {
 			return x + y
 		}
 
-		func getSomeInteger() { }
+		func getSomeInteger() { return }
```

Comment:
I think we need two tests anyway, one with explicit `return` and one without it.

Reply:
Fixed.